aws/resource_aws_kinesis_firehose_delivery_stream_test.go

```diff
@@ -153,7 +153,7 @@ func TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSE(t *testing.T) {
 	rInt := acctest.RandInt()
 	rName := fmt.Sprintf("terraform-kinesis-firehose-basictest-%d", rInt)
 	config := fmt.Sprintf(testAccKinesisFirehoseDeliveryStreamConfig_s3basic,
-		rInt, rInt, rInt, rInt)
+		rInt, rInt, rInt, rInt) + testAccKinesisFirehoseDeliveryStreamKmsKeyConfig(rName)
```

Comment:
Can you please create a new tests (one for AWS key and one for customer key) with their own test configurations rather than modifying existing testing? This is to ensure we are not introducing regressions. Copy-paste-modify of the testing is fine.

Reply:
done
```=== RUN   TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSE
=== PAUSE TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSE
=== RUN   TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSEAndCMK
=== PAUSE TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSEAndCMK
=== CONT  TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSE
=== CONT  TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSEAndCMK
--- PASS: TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSE (299.93s)
--- PASS: TestAccAWSKinesisFirehoseDeliveryStream_s3basicWithSSEAndCMK (322.85s)